tests/suites/test_suite_aria.function

```diff
@@ -0,0 +1,318 @@
+/* BEGIN_HEADER */
+#include "mbedtls/aria.h"
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_ARIA_C
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE */
+void aria_encrypt_ecb( char *hex_key_string, char *hex_src_string,
+                           char *hex_dst_string, int setkey_result )
+{
+    unsigned char key_str[1000];
+    unsigned char src_str[1000];
+    unsigned char dst_str[1000];
+    unsigned char output[1000];
+    mbedtls_aria_context ctx;
+    int key_len, data_len, i;
+
+    memset( key_str, 0x00, 1000 );
+    memset( src_str, 0x00, 1000 );
+    memset( dst_str, 0x00, 1000 );
+    memset( output, 0x00, 1000 );
+    mbedtls_aria_init( &ctx );
+
+    key_len = unhexify( key_str, hex_key_string );
+    data_len = unhexify( src_str, hex_src_string );
+
+    TEST_ASSERT( mbedtls_aria_setkey_enc( &ctx, key_str, key_len * 8 ) == setkey_result );
+    if( setkey_result == 0 )
+    {
+        for( i = 0; i < data_len; i += 16 )
+        {
+            TEST_ASSERT( mbedtls_aria_crypt_ecb( &ctx, MBEDTLS_ARIA_ENCRYPT, src_str + i, output + i ) == 0 );
+        }
+        hexify( dst_str, output, data_len );
+
+        TEST_ASSERT( strcasecmp( (char *) dst_str, hex_dst_string ) == 0 );
+    }
+
+exit:
+    mbedtls_aria_free( &ctx );
+}
+/* END_CASE */
+
+/* BEGIN_CASE */
+void aria_decrypt_ecb( char *hex_key_string, char *hex_src_string,
+                           char *hex_dst_string, int setkey_result )
+{
+    unsigned char key_str[1000];
+    unsigned char src_str[1000];
+    unsigned char dst_str[1000];
+    unsigned char output[1000];
+    mbedtls_aria_context ctx;
+    int key_len, data_len, i;
+
+    memset(  key_str, 0x00, 1000 );
```

Comment:
Consider using a single space (instead of a double) after `memset(` for style.

Reply:
Will fix.